src/cortex_cli/cortex_cli.py

```diff
@@ -537,6 +537,8 @@ def run(  #pylint: disable=too-many-arguments, too-many-locals, import-outside-t
         )
 
     logger.debug('\nResults:')
+    if results.metadata.calibration_set_id is not None:
+        logger.info('Using calibration set %d', results.metadata.calibration_set_id)
     logger.info(json.dumps(results.measurements[0]))  # pylint: disable=unsubscriptable-object
```

Comment:
Not directly related to this PR, but I think it's not right that the header `'\nResults:'` is printed with `logger.debug`, but the actual results are printed with `logger.info`.

Reply:
I thought it OK to have more verbose results formatting in debug mode. The header is not strictly needed, so leaving it out in non-verbose mode makes sense.